api/orchestrator/orchestrator.proto

```diff
@@ -192,7 +192,7 @@ service Orchestrator {
   }
 
   // Retrieves a metric configuration (target value and operator) for a specific
-  // service and metric ID
+  // service and metric ID. If no service ID is specified the default metric configuration is retrieved.
```

Comment:
That is not true. If It is so, then it is a bug. This call always needs a cloud service ID. If no specific metric configuration exists for the service, then the default one is returned, otherwise the "overwritten" one.